src/session-pool.ts

```diff
@@ -143,7 +143,6 @@ export interface SessionPoolOptions {
   max?: number;
   maxIdle?: number;
   min?: number;
-  writes?: number;
```

Comment:
Removing a property from an (exported) interface is a breaking change. We either need to do a major version bump and set up an LTS branch for the current version, or just deprecate the property without removing it.

Reply:
Good point. Let's keep it for now, and it can be removed next time for the major version bump.
I marked this field as deprecated.

Reply:
Can you update the comment above as well as it is used as part of automatically generated documentation

Reply:
Done.

Reply:
Yes, I updated it previously in https://github.com/googleapis/nodejs-spanner/pull/1739/files#diff-d6f67152c7ddf6462713d84c5314cab643222c347a9b1c87f59498d8cd9c84abR145